src/components/Search.js

```diff
@@ -89,11 +131,12 @@ export const Search = () => {
     }
   }, [consentGranted, isOnline])
 
-  const hideW = (searchResults || !isOnline || loading)
+  const hideW = (searchResults || !isOnline || loading || (isFeedExpanded && isTrendingArticlesGroup()))
```

Comment:
```suggestion
  const hideW = searchResults || !isOnline || loading || isFeedExpanded
```

I think this can be short like this, there is no need to double check if it is within the group, the `isFeedExpanded` is enough here